RELEASE-NOTES.md

```diff
@@ -1,3 +1,12 @@
+# Google Cloud IoT Core Device SDK for Embedded C version  1.0.2
+#### April 27, 2021
+
+- Updated the POSIX BSP to use open instead of fopen for more consistent error detection, [#100](https://github.com/GoogleCloudPlatform/iot-device-sdk-embedded-c/pull/100) by [@pnfisher}(https://github.com/pnfisher), thank you!
+- Fixed a non null-terminated issue in message.topic with ‘iotc_user_subscription_callback_’, [#111](https://github.com/GoogleCloudPlatform/iot-device-sdk-embedded-c/issues/111) by [@csobrinho](https://github.com/csobrinho), thank you!
+- Added overflow protection to ‘__iotc_calloc()’ calls.
```

Comment:
the two above have PR links, this could have PR link as well, but, I leave this up to you

Reply:
Done.